src/main/java/de/tarent/maven/plugins/pkg/upload/WagonUploader.java

```diff
@@ -3,21 +3,20 @@
 import java.io.File;
 
 import org.apache.maven.execution.MavenSession;
+import org.apache.maven.plugin.BuildPluginManager;
```

Comment:
The test needs to be changed to:

testConstructor(de.tarent.maven.plugins.pkg.upload.WagonUploaderTest): org.apache.maven.plugin.DefaultBuildPluginManager cannot be cast to org.apache.maven.plugin.PluginManager

Reply:
now all tests are green